src/elements/content-sidebar/versions/StaticVersionSidebar.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * @flow
+ * @file Versions Sidebar component
```

Comment:
Can we update the flow description to differentiate it from the main Versions Sidebar?

Reply:
Ack, will do